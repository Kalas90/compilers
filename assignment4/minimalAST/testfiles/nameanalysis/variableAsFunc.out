Error at line 3: le other error
