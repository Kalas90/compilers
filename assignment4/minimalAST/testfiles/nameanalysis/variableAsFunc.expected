Error at line 3: Tried to use variable as function handle.
